components/relevancy/src/lib.rs

```diff
@@ -159,8 +161,76 @@ impl RelevancyStore {
     /// its likelihood of a negative outcome.
     #[handle_error(Error)]
     pub fn bandit_update(&self, bandit: String, arm: String, selected: bool) -> ApiResult<()> {
+        let mut cache = self.cache.lock();
+
+        cache
+            .clear(&bandit, &arm)
+            .expect("failed to clear cached distribution");
```

Comment:
I commented below. `cache.clear()` doesn't fail, so no reason to return a `Result` and use `expect`. Also, expect means the application panics. We should return a `Error::CacheError` and let the calling application figure out how to handle an error case (logging or something else).